src/main/java/frc/robot/RobotContainer.java

```diff
@@ -45,7 +46,9 @@
 
   // The robot's subsystems and commands are defined here...
   private static final Joystick driverJoystick = new Joystick(Constants.portNumber0);
+  private static final Joystick driverJoystick2 = new Joystick(Constants.portNumber1);
   private JoystickButton[] buttons = new JoystickButton[13];
+  private JoystickButton[] buttons2 = new JoystickButton[13];
```

Comment:
Request better naming here too so it is easy to see which button collections are for which mapping.